### PR TITLE
add release notes for v1.0

### LINKED ISSUE #4451: Add release notes for v1.0 blog
Create release notes blog for v1.0

/assign

### PR DESCRIPTION
**Pull request by csantanapr**
Fixes #4451

### CODE REVIEW

**Comment by csantanapr** [comment]
/hold
need to add highlights

File: blog/docs/releases/announcing-knative-v1-0-release.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+    - [Apache Kafka Broker v1.0](#apache-kafka-broker-v10)
+    - [RabbitMQ Broker and Source v1.0](#rabbitmq-broker-and-source-v10)
+- `kn` [CLI v1.0](#client-v10)
+- [Knative Operator v1.0](#operator-v10)
+- [Thank you contributors](#thank-you-contributors)
+
+
+### Highlights
+
+- All components versions are now tag `knative-v1.0.0`, for users using go library use version `v0.27`
```
Comment (julz) [inline blog/docs/releases/announcing-knative-v1-0-release.md:34]:
```suggestion
- All components versions are now tagged `knative-v1.0.0`, for users using go modules use version `v0.27`
```

**Review by julz — COMMENTED** [review]
nit

File: blog/docs/releases/announcing-knative-v1-0-release.md (hunk 1 above)
Comment (csantanapr) [reply blog/docs/releases/announcing-knative-v1-0-release.md]: thank you I addressed the nit

**Pull request merged**